Include/Forest.h

```diff
@@ -11,7 +11,7 @@
 #include "Snowman.h"
 #include "Assets.h"
 
-#define ForestWidth 21
+#define ForestWidth 121
```

Comment:
i would suggest making this a bit smaller because it takes a while to load and the TA might think that our code doesnt work. Id put it maybe around 89?

Reply:
Yeah good point. On my PC it takes quite a while to compile (10 seconds) and my computer is pretty powerful

Reply:
Resolved! Optimized the generation